core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceInputCheckerServiceSpec.js

```diff
@@ -26,11 +26,11 @@
      $injector.get('MultipleChoiceInputCheckerService');
    }));
 
-   it('Should return true if interaction data is compatible', function(){
-     expect(multipleChoiceInputCheckerService.isValid()).toEqual(true);
+   it('Should return true if interaction data is compatible', function() {
+     expect(multipleChoiceInputCheckerService.isValid()).toBe(true);
    });
 
-   it('Should return flase if interaction data is not compatible', function(){
-     expect(multipleChoiceInputCheckerService.isValid()).toEqual(false);
+   it('Should return flase if interaction data is not compatible', function() {
+     expect(multipleChoiceInputCheckerService.isValid()).toBe(false);
```

Comment:
Similarly modify the values in stubs so that interaction data should not compatible and see if it returns False or not?